doc/user_guide/encodings/index.rst

```diff
@@ -44,6 +44,44 @@ the chart. For example, below we adjust the y-axis title and increase the step b
     )
 
 
+.. _method-based-attribute-setting:
+
+Alternative Syntax
+~~~~~~~~~~~~~~~~~~
+
+Altair 5.0 introduced an alternative method-based syntax for setting channel attributes.  In the example above, the ``axis`` attribute of the x channel encoding is set using the ``axis`` keyword argument: ``x=alt.X('Horsepower', axis=alt.Axis(tickMinStep=50))``.  To define the same :class:`X` object using the method-based syntax (new as of Altair 5.0), we can use ``x=alt.X('Horsepower').axis(tickMinStep=50)``.  In other words, the use of the ``axis`` keyword argument is replaced by the use of the ``axis`` method.
+
+The same technique works with all encoding channels and all channel attributes.  For example, notice how we make the analogous change with respect to the ``title`` attribute of the y channel.  The following produces the same chart as the previous example.
+
+.. altair-plot::
+    import altair as alt
+    from vega_datasets import data
+    cars = data.cars()
+
+    alt.Chart(cars).mark_point().encode(
+        x=alt.X('Horsepower').axis(tickMinStep=50),
+        y=alt.Y('Miles_per_Gallon').title("Miles per Gallon"),
+        color='Origin',
+        shape='Origin'
+    )
+
+These attribute-setter methods can also be chained together, as in the following, in which we set the ``axis``, ``bin``, and ``scale`` attributes of the x channel by using the corresponding methods (``axis``, ``bin``, and ``scale``).  We break the ``x`` definition over multiple lines to improve readability.  (This is valid syntax because of the enclosing parentheses from ``encode``.)
```

Comment:
```suggestion
These option-setter methods can also be chained together, as in the following, in which we set the ``axis``, ``bin``, and ``scale`` options of the ``x`` channel by using the corresponding methods (``axis``, ``bin``, and ``scale``).  We can break the ``x`` definition over multiple lines to improve readability.  (This is valid syntax because of the enclosing parentheses from ``encode``.)
```